great_expectations/data_context/config_validator/yaml_config_validator.py

```diff
@@ -331,9 +370,12 @@ def _test_yaml_config_prepare_config(
             )
             raise e
 
+    def _load_config_string_as_commented_map(
+        self, config_str: str, usage_stats_event_name: str
+    ) -> CommentedMap:
         try:
-            config: CommentedMap = yaml.load(config_str_with_substituted_variables)  # type: ignore[arg-type]
-            return config
+            substituted_config: CommentedMap = yaml.load(config_str)  # type: ignore[arg-type]
```

Comment:
`yaml.load` doesn't accept a string??

Reply:
hmm not sure why we have that `type: ignore` - I'll delete and see if it works.

Is there a way to identify outdated `type: ignore` comments?